content/getting-started/connect-the-pim-old/welcome.md

```diff
@@ -0,0 +1,33 @@
+# 👋 Welcome in this guide!
+
+This tutorial will teach you:
+> How to connect the PIM with the API on a v1.7, v2 or v3 PIM
+
+## Goal
+At the end of this tutorial, you will know what are the primary steps to properly connect the PIM.
+
+::: tips
+Connections will be achieved with the help of the API, as the API is the best way to connect the PIM. Wondering why? Discover the reasons [here](/documentation/introduction.html#why-should-you-use-our-api)!
+:::
+
+## Prerequisites
+
+<i class="fa fa-check-square"></i> **You have access to a a v1.7, v2 or v3 Akeneo PIM.**  
+This can seem pretty obvious. But yeah! You will need a PIM if you want to connect it to any third parties.  
+And of course, you also have to make sure that you can access it from where you are requesting.
+::: info
+This tutorial is dedicated to version of the PIM older than the v4. If you are working on a v4 PIM, we also got you covered! :wink: There is a tutorial for you right [here](/getting-started/connect-the-pim-4x/welcome.html).
+::: 
+
+<i class="fa fa-check-square"></i> **You already know to which third party you want to connect your PIM.**  
+It can be one of the following: an ERP, a DAM, an ecommerce platform, a syndication platform, a translation tool, a publishing tool... You name it! Pick one for which you need a connection with your PIM.
+
+As an example, in this tutorial, we will consider that we want to connect the PIM to your ERP.
+
+You will also have to know the use case for your connection.  
+In this tutorial, for our ERP example, let's say each day, new products are referenced into your ERP. You want those new products to be sent over to your PIM so that you can enrich them, in the PIM, the day after.
```

Comment:
```suggestion
In this tutorial, for our ERP example, let's say everyday day, new products are referenced into your ERP. You want those new products to be sent over to your PIM so that you can enrich them, in the PIM, the day after.
```

Reply:
everyday day

Reply:
```suggestion
In this tutorial, for our ERP example, let's say every day, new products are referenced into your ERP. You want those new products to be sent over to your PIM so that you can enrich them, in the PIM, the day after.
```